pages/common/czkawka-cli.md

```diff
@@ -0,0 +1,16 @@
+# czkawka-cli
+
+> Multi functional app to find duplicates, empty folders, similar images etc. Command-line version.
+> More information: <https://github.com/qarmin/czkawka>.
+
+- Find [dup]licate files in directory:
+
+`czkawka-cli dup --directories {{path/to/directory}}`
```

Comment:
```suggestion
- List duplicate or similar files in specific directories:

`czkawka-cli {{dup|image}} --directories {{path/to/directory1 path/to/directory2 ...}}`
```

- Optional fix (multiple occurrences): `list` may sound better as it's not clear what is going to be with search results with the current description.
- Optional fix (multiple occurrences): `specific` may sound better.
- Optional fix (multiple occurrences): mnemonics can be not used with long options.
- Optional fix: this example can be joined with the last one.
- Optional fix (multiple occurrences): example can be generalized.